library/swift/src/GRPCClient.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+/// Client that supports sending and receiving gRPC traffic.
+@objcMembers
+public final class GRPCClient: NSObject {
+  private let client: Client
```

Comment:
Not directly related to this PR but per offline discussion, suggest renaming `Client` to something a little more descriptive, e.g. `HTTPClient`.

Reply:
Yep, good callout. Updating here: https://github.com/lyft/envoy-mobile/pull/425